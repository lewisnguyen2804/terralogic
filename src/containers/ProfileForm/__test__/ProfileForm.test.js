import React from 'react'
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import { ProfileForm } from '../';

describe('test for Profile Page', () => {
    const wrapper = shallow(<ProfileForm />);
    it('should render correctly', () => {
        expect(JSON.stringify(wrapper)).toMatchSnapshot();
    });
});

// handleSubmit function
// describe('test for Profile Page', () => {
    // const wrapper = shallow(<RegisterForm />);
    // const instance = wrapper.instance();
    // const registerClick = jest.spyOn(instance, 'handleSubmit');
    // wrapper.update();
    // instance.forceUpdate();
    // it('it should render with check func', () => {
    //     wrapper.find('form').simulate('submit', { preventDefault() { } });
    //     expect(registerClick).toHaveBeenCalled();
    // });
// });
