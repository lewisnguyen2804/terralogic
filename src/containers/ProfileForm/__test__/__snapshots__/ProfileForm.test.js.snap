// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for Profile Page should render correctly 1`] = `"{}"`;
